app/jobs/import_versions_job.rb

```diff
@@ -47,7 +47,9 @@ def import_raw_data(raw_data)
   end
 
   def import_record(record)
-    page = find_or_create_page_for_record(record)
+    page = page_for_record(record, create: @import.create_pages)
+    return unless page
```

Comment:
Does it seem potentially useful to issue a log message when a `url` is skipped here? That might be too verbose, depending on how often this happens in practice. Just a thought.

Reply:
How about a separate list, processing_warnings?

The endpoints also seem like a very good idea but not necessarily a blocker for the PR.

Reply:
Does 70641ec8a75ce849219ed961d255e5566cf3ace8 seem like a reasonable approach?